charts/garnet/README.md

```diff
@@ -10,6 +10,23 @@ A Helm chart for Microsoft garnet
 
 * <https://github.com/microsoft/garnet.git>
 
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts. Please refer to
+Helm's [documentation](https://helm.sh/docs) to get started.
+
+To install the garnet chart (using an OCI-based registry):
+
+```sh
+helm upgrade --install garnet oci://ghcr.io/microsoft/helm-charts/garnet
+ ```
+
+To uninstall the chart:
+
+```sh
+helm delete garnet
+```
+
```

Comment:
> "--port","6379","-m","128m","-i","128m"

I dont think these arguments are necessary any more as they default to proper values.

Reply:
Fixed by 1cd1e40c4b3f0111b27724dcde8b68c25dfc9cf0